src/utils/tests/builders/evm/transaction.rs

```diff
@@ -0,0 +1,105 @@
+use crate::models::{EVMBaseTransaction, EVMTransaction};
+use alloy::primitives::{Address, Bytes, B256, U256};
+
+/// A builder for creating test EVM transactions with default values.
+#[derive(Debug, Default)]
+pub struct TestTransactionBuilder {
```

Comment:
For consistency, can we rename this to `TransactionBuilder` (see MonitorBuilder, NetworkBuilder, TriggerBuilder)?

Reply:
sure